python/integrationtest/integrationtest_drunc.py

```diff
@@ -366,7 +366,9 @@ class RunResult:
     )
 
     if connsvc_obj is not None:
+        time.sleep(1)
         connsvc_obj.send_signal(2)
+        time.sleep(0.5)
```

Comment:
This is probably fine, but it may be interesting to measure the amount of time that is usually needed for a clean shutdown upon receiving signal 2, and set this time to 110% of that usual time...